docs/source/user/accessibility.rst

```diff
@@ -0,0 +1,112 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The JupyterLab interface allows users to cycle through focusable elements using the :kbd:`Arrow keys`, :kbd:`Tab key` and :kbd:`Shift key`. This sequential navigation allows the exploration of buttons, input fields, and interactive components in a given order.
+
+In the main UI sections, including the notebook, console, sidebars and terminal, it is possible for users to become trapped within a specific area, limiting keyboard navigation. This is a work in progress and multiple efforts are being made to reduce the number of tab traps.
+
+In case of encountering a Tab trap, users can use the Command Palette. By opening the Command Palette and searching for the desired command, this action enables the transition to other parts of JupyterLab's interface without relying solely on sequential tabbing.
+
+Colour Themes
+^^^^^^^^^^^^^
+
+While JupyterLab provides default themes, it's important to note that these themes are not compliant with the `Web Content Accessibility Guidelines (WCAG) 2.0 <https://www.w3.org/TR/WCAG20/>`_. This means that some users might experience challenges in terms of colour contrast or readability.
+
+Nevertheless, `Jupyterlab Theme Editor <https://github.com/jupyterlab-contrib/jupyterlab-theme-editor>`_, and `Jupyterlab Accessible Themes <https://github.com/Quansight-Labs/jupyterlab-accessible-themes>`_, a third-party extension provides an additional avenue for improving color accessibility in JupyterLab. This extension introduces two WCAG-compliant accessible themes to the JupyterLab ecosystem. Please note that this extension is compatible with JupyterLab 3.0.
+
+In addition to accessible third-party themes, JupyterLab helps users to create and apply custom themes. There are plenty of resources for developing a new theme including `this detailed blog post
+<https://labs.quansight.org/blog/2020/12/jupyterlab-winter-theme>`_ and
+the `Extension Development Guide
+<https://jupyterlab.readthedocs.io/en/latest/extension/extension_dev.html>`_
+
+Text Scaling
+^^^^^^^^^^^^
+
+JupyterLab allows text scaling customization via the advanced settings for improving the legibility of certain parts of the interface like files. This improves the experience for users with visual impairment. For changing the font size options, please navigate via the main menu to
+`Settings > Advanced Settings` and locate the text editor settings on the left panel to make the necessary adjustments.
+
+It's important to note that changes made in this setting will exclusively impact either the Markdown viewer or the text editor, based on the selected configuration. Notably, adjustments here won't influence the font size in the notebook area or in the rest of the interface.
+
+However, users have the zoom browser functionality to increase the size of the interface elements of JupyterLab. This approach helps to improve readability in specific contexts. Take into account that the limit for font size is 100%.
```

Comment:
Zoom browsers can go up to 400%, where did you found that the limit for font size is 100%?